source/website/src/views/Step4.vue

```diff
@@ -38,8 +38,8 @@ SPDX-License-Identifier: Apache-2.0
             </b-row>
             <b-row>
               <b-col cols="7">
-                <h5>Input file:</h5>
-                {{ "s3://" + DATA_BUCKET_NAME + "/" + s3key }}
+                <h5>Input files from {{"s3://" + DATA_BUCKET_NAME}}:</h5>
+                {{ s3key }}
```

Comment:
Should we list all of the selected files here?

Reply:
They should all be displayed as inputted in a comma delimited string - do you think a bullet point list would be better? I chose to leave it as a string to reduce any latency from transforming the comma delimited string to an array